platforms/ios/lib/WysiwygComposer/Tests/WysiwygComposerTests/Components/WysiwygComposerView/WysiwygComposerViewModelTests+Suggestions.swift

```diff
@@ -116,7 +116,7 @@ extension WysiwygComposerViewModelTests {
         XCTAssertEqual(
             viewModel.content.html,
             """
-            <a data-mention-type="room" contenteditable="false" href="https://matrix.to/#/#room1:matrix.org">Room 1</a> Text
```

Comment:
Due note the space removal in those cases, maybe from a UX perspective we should add a space after the mention when we insert it without a suggestion 🤔 (which was the previous behaviour for this on the iOS impl)

Reply:
I think that the check we do in Rust probably needs to be a bit better than it is. Currently it just looks for a next sibling and, if there isn't one, adds a space. I don't feel this is sufficient.

Perhaps it's easier to define when we don't add a space. To me it seems that we shouldn't add a space if there is a leaf following the mention we've just added _and_ that leaf doesn't start with a space. In all other cases add a space. Does that sound closer to what we want? It's quite hard trying to imagine all the cases of mention next to X element.

Reply:
Yep, got it the wrong way around, will edit